src/dashboard/src/components/api/views.py

```diff
@@ -858,3 +865,65 @@ def validate(request, validator_name):
             "Unexepected error in the validation process, see the logs for more details."
         )
     return _ok_response({"valid": True})
+
+
+def format_datetime(value):
+    try:
+        return value.strftime("%Y-%m-%dT%H:%M:%S")
+    except AttributeError:
+        pass
+
+
+def format_task(task, detailed_output=False):
+    result = {"uuid": task.taskuuid, "exit_code": task.exitcode}
+    if detailed_output:
+        result.update(
+            {
+                "file_uuid": task.fileuuid,
+                "file_name": task.filename,
+                "time_created": format_datetime(task.createdtime),
+                "time_started": format_datetime(task.starttime),
+                "time_ended": format_datetime(task.endtime),
+                "duration": helpers.task_duration_in_seconds(task),
+            }
+        )
+    return result
+
+
+@_api_endpoint(expected_methods=["GET"])
+def unit_jobs(request, unit_uuid):
+    jobs = models.Job.objects.filter(sipuuid=unit_uuid)
```

Comment:
```suggestion
    jobs = models.Job.objects.filter(sipuuid=unit_uuid).order_by("createdtime")
```
This might be a good way to provide some predictability to the structure returned that can be used by the consumer where we don't provide the `datetime` to work with initially. I could imagine the result of this AIP call being used to create a timeline of jobs other than we replay for the user in the dashboard.

Reply:
:+1: